var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -90,8 +91,8 @@ class Seacas(CMakePackage):
     depends_on('adios2@develop~mpi', when='+adios2 ~mpi')
     depends_on('adios2@develop+mpi', when='+adios2 +mpi')
     depends_on('matio', when='+matio')
-    depends_on('metis+int64+real64', when='+metis ~mpi')
-    depends_on('parmetis+int64+real64', when='+metis +mpi')
+    depends_on('metis+int64+real64', when='+metis')
```

Comment:
Why did you drop `~mpi`? Do you want both metis and parmetis when `+mpi`?

Reply:
Yes, metis is always a dependency.  If `+mpi`, then parmetis is a dependency which then depends on metis.